.eslintrc

```diff
@@ -0,0 +1,79 @@
+{
+  "parser": "babel-eslint",
```

Comment:
I found out from the ESLint folks that babel-eslint is no longer necessary. We just need to add `"es6": true` to the `"env"` object.